tests/page-objects/login/login.po.js

```diff
@@ -47,5 +59,40 @@ module.exports = {
       await browser.wait(() => helper.isTextDisplayed(incorrectCredentialsText), 2000);
     }
   },
-  returnToLogin: () => element(by.css('.btn[href="/medic/login"]'))
+  returnToLogin: () => element(by.css('.btn[href="/medic/login"]')),
+  
+  getAllLocales: async () => {
+    const locales = await element.all(by.css('.locale')).map(loc => {
+      return {
+        code: loc.getAttribute('name'),
+        name: loc.getText(),       
+      };
+    });
+    return locales;
+  },
+  
+  getLabelForPassword,
+  labelForUser: async () => await helper.getTextFromElementNative(getLabelForUser),
+  labelForPassword: async () => await helper.getTextFromElementNative(getLabelForPassword),
+  getselectedLanguage: async () => {
+    const lang = await element(by.css('.locale.selected')).map(loc => {
+      return {
+        code: loc.getAttribute('name'),
+        name: loc.getText(),       
+      };
+    });
+    return lang;
+  },
+
+  changeLanguage: async (code) => {
+    await changeLocale(code);
+    await getLoginButton().click();
+    await browser.sleep(10000);
```

Comment:
Can we remove the sleep here? Is the next line `await helper.waitUntilReadyNative(getselectedLanguage());` not enough to know the test can move on?

Reply:
yeah - to be removed. I only use sleep when debugging, shouldn't be in production